step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/aggregation/TimeSeriesAggregationQuery.java

```diff
@@ -0,0 +1,102 @@
+package step.core.timeseries.aggregation;
+
+import step.core.collections.Filter;
+import step.core.timeseries.TimeSeries;
+import step.core.timeseries.query.TimeSeriesQuery;
+
+import java.util.*;
+import java.util.function.Function;
+
+public class TimeSeriesAggregationQuery extends TimeSeriesQuery {
```

Comment:
@lucianbaciu This class TimeSeriesAggregationQuery.java. Use the refactor feature of the IDE to rename this class.